pom.xml

```diff
@@ -80,7 +80,7 @@
   	<dependency>
   		<groupId>com.azure</groupId>
   		<artifactId>azure-cosmos</artifactId>
-  		<version>4.11.0</version>
+  		<version>4.47.0</version>
```

Comment:
Instead of a version, can we use the version `LATEST`? which will ensure it is always up to date, like we have in samples - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples/blob/main/pom.xml#L48

Reply:
changed to LATEST